homeassistant/components/intellifire/entity.py

```diff
@@ -5,23 +5,25 @@
 from homeassistant.helpers.entity import EntityDescription
 from homeassistant.helpers.update_coordinator import CoordinatorEntity
 
-from . import IntellifireDataUpdateCoordinator
+from . import IntelliFireDataUpdateCoordinator
 
 
-class IntellifireEntity(CoordinatorEntity[IntellifireDataUpdateCoordinator]):
-    """Define a generic class for Intellifire entities."""
+class IntellifireEntity(CoordinatorEntity[IntelliFireDataUpdateCoordinator]):
+    """Define a generic class for IntelliFire entities."""
 
     _attr_attribution = "Data provided by unpublished Intellifire API"
     _attr_has_entity_name = True
 
     def __init__(
         self,
-        coordinator: IntellifireDataUpdateCoordinator,
+        coordinator: IntelliFireDataUpdateCoordinator,
         description: EntityDescription,
     ) -> None:
         """Class initializer."""
         super().__init__(coordinator=coordinator)
         self.entity_description = description
-        self._attr_unique_id = f"{description.key}_{coordinator.read_api.data.serial}"
+        self.identifiers = ({("IntelliFire", f"{coordinator.fireplace.serial}]")},)
```

Comment:
why?

Reply:
Its down below -> the value is now coming from the coordinator instead of coordinator.read_api.data